### PR TITLE
Feat add volcano embedding

### PR DESCRIPTION
**Pull request by lujingxuansc**
Description: Volcano Ark is an enterprise-grade large-model service platform for developers, providing a full range of functions and services such as model training, inference, evaluation, fine-tuning. You can visit its homepage at https://www.volcengine.com/docs/82379/1099455 for details. This change could help developers use the platform for embedding.
Issue: None
Dependencies: volcengine
Tag maintainer: @baskaryan
Twitter handle: @hinnnnnnnnnnnns

### CODE REVIEW

File: libs/langchain/langchain/embeddings/__init__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from langchain.embeddings.baidu_qianfan_endpoint import QianfanEmbeddingsEndpoint
 from langchain.embeddings.bedrock import BedrockEmbeddings
 from langchain.embeddings.bookend import BookendEmbeddings
+from langchain.embeddings.bytedance_volcano import VolcanoEmbeddings
```
Comment (baskaryan) [inline libs/langchain/langchain/embeddings/__init__.py:26]: no need to do anything in `langchain`. existing imports are kept for backwards compatibility.

Comment (lujingxuansc) [reply libs/langchain/langchain/embeddings/__init__.py]: fixed

**Comment by lujingxuansc** [comment]
PTAL @baskaryan

**Comment by lujingxuansc** [comment]
@baskaryan Sorry to bother you, could you please review and merge my PR? Thanks!

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**